Assets/MixedRealityToolkit/Inspectors/Profiles/BaseMixedRealityToolkitConfigurationProfileInspector.cs

```diff
@@ -244,5 +245,50 @@ protected void CheckMixedRealityInputActions()
                 }
             }
         }
+
+        /// <summary>
+        /// Applies the most relevant settings of DataProviderConfigurations
+        /// </summary>
+        /// <param name="typeName">name of the Type of the Data Provider</param>
+        /// <param name="providerAttribute"> the corresponding attribute</param>
+        /// <param name="providerName"> the target provider Name</param>
+        /// <param name="configurationProfile">the target configuration profile</param>
+        /// <param name="runtimePlatform">the target runtime plaform</param>
+        protected void ApplyDataProviderConfiguration(
+            string typeName,
```

Comment:
why pass the string name? Just pass the type

Reply:
I have passed the name because I need it in the calling code a little further down too. passing the type requires me to either return the name or search for the type a second time